docs/meshcloud.project.md

```diff
@@ -3,42 +3,56 @@ id: meshcloud.project
 title: Project
 ---
 
-In order to use cloud computing resources, you have to create a meshProject. Within the meshProject, you can manage and scale the resources you use across all locations.
+In order to use cloud computing resources, you have to create a meshProject. Within the meshProject, you can manage and scale the resources you use across all locations. Only Customer Admins have access to the administrative functionality described in this section.
 
 ## Create a new meshProject
 
 Log on to the Meshpanel with your account credentials and press `Create`. Type in a name for your new project and press `Next`. Choose a payment method as well as your billing address and press `Next` again. In the last step, you can choose the locations from which you can obtain the cloud computing resources for your project. Press `Create Project` and you will see your new project listed in the Projects List.
 
-## User Management on a meshProject
+## Manage meshProjects
 
-Before being able to assign a user to a meshProject, the user must have been [invited to the Meshcustomer](meshcloud.customer.md). Then go to your **Account** settings. You access them by pressing the settings icon on the top right of the panel. From here, navigate to **Projects** and click the pen button to edit the project. In the **Project Users** section all users of the project are listed and new one can be added.
+In your **Account** section under **Projects** all meshProjects of your meshCustomer are listed. You find the **Account** section by pressing the settings icon on the top right of the mesPanel. You can execute administrative actions on your meshProjects from the **Projects** section. To edit a project, click the **pen** icon in the **Actions** column of the meshProject list. You can edit the assigned locations, payment information and the assigned users of the project.
 
-### Assign user to a meshProject
+The following diagram shows how access rights and project replication result in the correct project access for your users.
+
+![Project User Role Replication](assets/project-user-roles.png)
+
+Project Users always get access to all cloud platforms, that are assigned to the project.
```

Comment:
Would not write users in upper case letters as its not a special "type"?

Reply:
I am also not sure how to handle this. Project User is some kind of special type, but also not definetly a special type. I talked to CHristina and she says, we write these pseudo types in lowercase letters. Only our main entites shall be emphasized, which we do by writing meshCustomer, meshProject, meshUser, etc